rts/Sim/Units/CommandAI/BuilderCaches.cpp

```diff
@@ -0,0 +1,147 @@
+#include "Sim/Units/CommandAI/BuilderCaches.h"
```

Comment:
Probably needs the licence notice at the top (as Spring, since it's entirely old code)
```suggestion
/* This file is part of the Spring engine (GPL v2 or later), see LICENSE.html */
#include "Sim/Units/CommandAI/BuilderCaches.h"
```